com.unity.ml-agents.extensions/Runtime/Match3/Match3Actuator.cs

```diff
@@ -9,17 +9,14 @@ namespace Unity.MLAgents.Extensions.Match3
     /// Actuator for a Match3 game. It translates valid moves (defined by AbstractBoard.IsMoveValid())
     /// in action masks, and applies the action to the board via AbstractBoard.MakeMove().
     /// </summary>
-    public class Match3Actuator : IActuator, IHeuristicProvider, IBuiltInActuator
+    public class Match3Actuator : IActuator, IBuiltInActuator
     {
-        protected AbstractBoard m_Board;
-        protected System.Random m_Random;
+        private AbstractBoard m_Board;
+        private System.Random m_Random;
         private ActionSpec m_ActionSpec;
         private bool m_ForceHeuristic;
         private Agent m_Agent;
-
-        private int m_Rows;
-        private int m_Columns;
-        private int m_NumCellTypes;
+        private BoardSize m_MaxBoardSize;
```

Comment:
Nit: `private` is redundant

Reply:
Yeah, Rider user to complain about it, but now it seems to want to add it all the time.

Reply:
Removed all the `private`s here (and on Match3Sensor)